packages/smooth_app/lib/pages/product/edit_ocr_page.dart

```diff
@@ -83,9 +89,7 @@ class _EditOcrPageState extends State<EditOcrPage> with UpToDateMixin {
       if (_controller.text != extractedText) {
         setState(() => _controller.text = extractedText);
       }
-    } catch (e) {
-      //
-    }
+    } catch (_) {}
```

Comment:
So you're not resetting `_extractingData` to `false` when there's an exception in `getExtractedText`?

Reply:
Nice catch 😆
I will fix that